c-gull/src/lib.rs

```diff
@@ -14,14 +15,21 @@ pub use libc::*;
 #[macro_use]
 mod use_libc;
 
+#[cfg(feature = "std")]
 mod nss;
+#[cfg(feature = "std")]
 mod printf;
+#[cfg(feature = "std")]
 mod strtod;
+#[cfg(feature = "std")]
 mod strtol;
+#[cfg(feature = "std")]
 mod time;
 
+#[cfg(feature = "std")]
 use std::ffi::CStr;
```

Comment:
This `CStr` import could be moved inside of `__asert_fail`'s body to eliminate a `cfg`.